src/pages/guide/language/overview.md

```diff
@@ -3,6 +3,18 @@ title: Overview
 order: 0
 ---
 
+```playground:reason
+let rec hanoi n a b c =>
+  if (n > 0) {
+    hanoi (n - 1) a c b;
+    Js.log {j|Move disk from pole $a to pole $b|j};
+    hanoi (n - 1) c b a
+  };
+  
+let () =
+  hanoi 4 1 2 3;
+```
+
 Primitive                             | Example
 --------------------------------------|--------------------------------
 Strings                               |  `"Hello"`
```

Comment:
This page needs more update. It's a rather weak table for an overview

Reply:
I'll fill it in e a bit more before this is merged.

Reply:
Also, feel free to add to this PR if there's something you'd like to have here.

Reply:
Ah don't worry about this one, I'll change it myself at one point